### ISSUE TITLE
Uncaught TypeError: t is undefined

### ISSUE THREAD

**Issue by okeowosv** [edisonneza/jspdf-invoice-template#33]
I followed the installation correctly but still have this issue of **Uncaught TypeError: t is undefined [index.js:2:340710](https://unpkg.com/jspdf-invoice-template@1.3.1/dist/index.js)** below is my code 

```
const generatePdf = () => {
  var pdfObject = jsPDFInvoiceTemplate.default(props);
  console.log(pdfObject)
  var props = {
    outputType: jsPDFInvoiceTemplate.OutputType.Save,
    returnJsPDFDocObject: true,
    fileName: "Invoice 2021",
    orientationLandscape: false,
    logo: {
        src: "https://raw.githubusercontent.com/edisonneza/jspdf-invoice-template/demo/images/logo.png",
        type: 'PNG', //optional, when src= data:uri (nodejs case)
        width: 53.33, //aspect ratio = width/height
        height: 26.66,
        margin: {
            top: 0, //negative or positive num, from the current position
            left: 0 //negative or positive num, from the current position
        }
    },
    stamp: {
        inAllPages: true, //by default = false, just in the last page
        src: "https://raw.githubusercontent.com/edisonneza/jspdf-invoice-template/demo/images/qr_code.jpg",
        type: 'JPG', //optional, when src= data:uri (nodejs case)
        width: 20, //aspect ratio = width/height
        height: 20,
        margin: {
            top: 0, //negative or positive num, from the current position
            left: 0 //negative or positive num, from the current position
        }
    },
    business: {
        name: "Business Name",
        address: "Albania, Tirane ish-Dogana, Durres 2001",
        phone: "(+355) 069 11 11 111",
        email: "email@example.com",
        email_1: "info@example.al",
        website: "www.example.al",
    },
    contact: {
        label: "Invoice issued for:",
        name: "Client Name",
        address: "Albania, Tirane, Astir",
        phone: "(+355) 069 22 22 222",
        email: "client@website.al",
        otherInfo: "www.website.al",
    },
    invoice: {
        label: "Invoice #: ",
        num: 19,
        invDate: "Payment Date: 01/01/2021 18:12",
        invGenDate: "Invoice Date: 02/02/2021 10:17",
        headerBorder: false,
        tableBodyBorder: false,
        header: [
          {
            title: "#", 
            style: { 
              width: 10 
            } 
          }, 
          { 
            title: "Title",
            style: {
              width: 30
            } 
          }, 
          { 
            title: "Description",
            style: {
              width: 80
            } 
          }, 
          { title: "Price"},
          { title: "Quantity"},
          { title: "Unit"},
          { title: "Total"}
        ],
        table: Array.from(Array(10), (item, index)=>([
            index + 1,
            "There are many variations ",
            "Lorem Ipsum is simply dummy text dummy text ",
            200.5,
            4.5,
            "m2",
            400.5
        ])),
        additionalRows: [{
            col1: 'Total:',
            col2: '145,250.50',
            col3: 'ALL',
            style: {
                fontSize: 14 //optional, default 12
            }
        },
        {
            col1: 'VAT:',
            col2: '20',
            col3: '%',
            style: {
                fontSize: 10 //optional, default 12
            }
        },
        {
            col1: 'SubTotal:',
            col2: '116,199.90',
            col3: 'ALL',
            style: {
                fontSize: 10 //optional, default 12
            }
        }],
        invDescLabel: "Invoice Note",
        invDesc: "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary.",
    },
    footer: {
        text: "The invoice is created on a computer and is valid without the signature and stamp.",
    },
    pageEnable: true,
    pageLabel: "Page ",
};
}


```

**Comment by okeowosv** [comment]
![Uploading console.PNG…]()

**Comment by okeowosv** [comment]
![console](https://user-images.githubusercontent.com/51259940/199731403-f6461265-43fb-4278-a794-f3f35d6d7d23.PNG)

**Comment by edisonneza** [comment]
Can u try to declare the props object before calling the method?

**Comment by okeowosv** [comment]
> Owner

Appreciate the quick response but now I hav
![ssss](https://user-images.githubusercontent.com/51259940/199733842-0469e2ba-e325-4b21-872a-b1504efcec9c.PNG)
e the error below

**Comment by okeowosv** [comment]
> > Owner
> 
> Appreciate the quick response but now I hav ![ssss](https://user-images.githubusercontent.com/51259940/199733842-0469e2ba-e325-4b21-872a-b1504efcec9c.PNG) e the error below

Never mind i solved it but using the latest cdn v1.4.1 I was using v1.3.1 based on a tutorial I was following, many thanks.

**Comment by edisonneza** [comment]
Great that works now!
